queue-workers/queue.js

```diff
@@ -1,14 +1,29 @@
-const { Queue } = require('bullmq');
-const config = require('./config');
-const constants = require('../utils/constants');
 
-const queueConfig = config.get();
-queueConfig.defaultJobOptions = { attempts: 2, backoff: 1000 };
+const { QueueClient } = require("@azure/storage-queue");
+const azure_storage_connection_string = process.env.AZURE_STORAGE_CONNECTION_STRING;
+const queueName = process.env.QUEUE_NAME;
+const queueClient = new QueueClient(azure_storage_connection_string, queueName);
 
-const queue = new Queue(constants.CHATBOX_QUEUE_NAME, queueConfig);
 
 module.exports = {
-  add(name, data, opts) {
-    return queue.add(name, data, opts);
+  async sendMessage(name, data) {
+    try {
+      if (!data || !name) throw Error('No data/name found');
+      let payload = {
+         name,
+         data
+      };
+
+      payload = JSON.stringify(payload);
+
+      const sendMessage = await queueClient.sendMessage(payload);
+      return {
+        message: 'Action successfully to job queue',
```

Comment:
Could we just return `sendMessage.messageId`. 
It seems from `services/functions/chat-box.js` I don't think you are using the message property
You could then update the `job` constant in chat-box service to` jobId`